src/navigation/Nav.jsx

```diff
@@ -409,6 +415,13 @@ export class Nav extends React.Component {
 				className: `atMedium_display--none ${CLASS_AUTH_ITEM}`,
 				icon: <Icon shape="groups" size="s" className="display--block" />,
 			},
+			isProInNavDesktop && {
```

Comment:
Can you add a new story with pro in nav button to `nav.story.jsx`?

Reply:
Very confused on how to add this specific scenario. 

![Screenshot 2023-01-20 at 2 56 55 PM](https://user-images.githubusercontent.com/49497414/213804291-eecbf0f6-7398-42ff-a0e0-6d1efde12d43.png)

Reply:
You have mistake in the variable name. I told you to fix `trypPro`, but you didn't do it and just closed the comment. You have `trypPro` in Nav.jsx and `tryPro` in nav.story.jsx. It should be `tryPro` in both places.

Reply:
Did not notice the extra 'p'